src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,34 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(SessionFactory sessionFactory) {
+        actorDao = new ActorDaoImpl(sessionFactory);
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        try {
+            return actorDao.add(actor);
+        } catch (DataProcessingException e) {
+            return null;
+        }
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        try {
+            return actorDao.get(id).orElse(null);
```

Comment:
Don’t return null from the method (fix it in other methods)
```suggestion
            return actorDao.get(id)
                 .orElseThrow(() -> new DataProcessingException(
                        "Actor with id=%d was not found".formatted(id))
                );
```